### ISSUE TITLE
Output full SRI hashes (include `sha256-` prefix)

### ISSUE THREAD

**Issue by sersorrel** [seppeljordan/nix-prefetch-github#49]
Currently nix-prefetch-github's output contains a raw base64'd sha256 hash:

```console
$ nix-prefetch-github seppeljordan nix-prefetch-github
{
    "owner": "seppeljordan",
    "repo": "nix-prefetch-github",
    "rev": "50b4ccc03b03cc963d329e475f2b4c5ebf1dd178",
    "sha256": "YobBihNPbqYYWhe3x0p+BIlEK8R62s/dDFWUzP7fCTI=",
    "fetchSubmodules": false,
    "leaveDotGit": false,
    "deepClone": false
}
```

It would be ideal if nix-prefetch-github could include the SRI-style `sha256-` prefix in the hash, like `sha256-YobBihNPbqYYWhe3x0p+BIlEK8R62s/dDFWUzP7fCTI=`; this makes it clearer which kind of hash it is and more compatible with other tools (e.g. `nix hash to-base32 YobBihNPbqYYWhe3x0p+BIlEK8R62s/dDFWUzP7fCTI=` fails with `error: hash 'YobBihNPbqYYWhe3x0p+BIlEK8R62s/dDFWUzP7fCTI=' does not include a type, nor is the type otherwise known from context`).

**Comment by hraban** [comment]
Related: a registry pin looks like this:

```
{
  "flakes": [
    {
      "from": {
        "id": "nixpkgs",
        "type": "indirect"
      },
      "to": {
        "lastModified": 1682704613,
        "narHash": "sha256-HPzPRFiy2o/7k7mtnwfM1E6NVZHiFbPdmYCMoIpkHO4=",
        "owner": "NixOS",
        "repo": "nixpkgs",
        "rev": "8e3b64db39f2aaa14b35ee5376bd6a2e707cadc2",
        "type": "github"
      }
    }
  ],
  "version": 2
}
```

that would be a possible format to follow

**Comment by SuperSandro2000** [comment]
This is behavior sentimentally wrong and shouldn't be used. It makes it harder to migrate the hashes from sha256 to hash and technically the sri hash is incomplete.

**Comment by seppeljordan** [comment]
Implemented as of version 7.0.0
